pkg/kubelet/kubelet_volumes.go

```diff
@@ -127,8 +129,23 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*v1.Pod, runningPods []*kubecon
 			continue
 		}
 		if len(volumePaths) > 0 {
-			orphanVolumeErrors = append(orphanVolumeErrors, fmt.Errorf("Orphaned pod %q found, but volume paths are still present on disk", uid))
-			continue
+			for _, volumePath := range volumePaths {
```

Comment:
Can we move this code into a function of its own? Something like - `cleanupVolumePaths(volumePaths) error` . The idea is - if this function returns error then we can call continue and get rid of continue label which is bit hard to read.